src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -1208,7 +1234,19 @@ codet java_bytecode_convert_methodt::convert_instructions(
     else if(statement=="invokedynamic")
     {
       // not used in Java
-      code_typet &code_type=to_code_type(arg0.type());
+      code_typet &code_type = to_code_type(arg0.type());
+
+      const optionalt<symbolt> &lambda_method_symbol = get_lambda_method_symbol(
+        lambda_method_handles,
+        code_type.get(ID_java_lambda_method_handle_index));
```

Comment:
Probably better to use `get_int` here and have `get_lambda_method_symbol` take an `int` as this handles the conversion for you.